app/src/main/java/com/hyperether/getgoing/ui/fragment/ProfileFragment.java

```diff
@@ -280,7 +283,6 @@ private void initDialogs() {
                     SharedPreferences.Editor editor = settings.edit();
```

Comment:
@zukovicfilip Create separate class SharedPref with static methods, and move all SharedPreferences manipulation out of views. So that class can have methods like: SharedPref.setHeight(value); and int values = SharedPref.getHeight();